apps/meteor/app/apps/client/orchestrator.js

```diff
@@ -160,6 +160,8 @@ class AppClientOrchestrator {
 		return effectiveStatus;
 	};
 
+	screenshots = async (appId) => APIClient.get(`apps/${appId}/screenshots`);
```

Comment:
```suggestion
	screenshots = (appId) => APIClient.get(`apps/${appId}/screenshots`);
```

Since you are not using the `await` inside of this function, you can remove this `async`.